### ISSUE TITLE
deeplink / anchors not working on firefox IOS?

### ISSUE THREAD

**Issue by simkne** [alvarotrigo/pagePiling.js#184]
Hi.
First, thanks alot for pagepile, excelent piece of code..
I was just wondering if anyone else has experienced that deeplinks / Anchors are not working on firefox..
check this article out I made with pagepiling.js
somehow I can deeplink on safari and chrome, but it isn't working on firefox:
https://www.addendum.org/atomkraft/atomstrom/#these8

Any ideas?
cheers,
Simon

**Comment by alvarotrigo** [comment]
They work as expected on the [demo page](https://alvarotrigo.com/pagePiling/).

**Comment by simkne** [comment]
you are right. weird, can't figure out what I did wrong then. it is working on all browsers except firefox...

**Comment by alvarotrigo** [comment]
Consider perhaps using instead my other libarry, fullPage.js:
https://alvarotrigo.com/fullPage/

**Comment by simkne** [comment]
tricky... it just went live...
You like what I did with your Library ? ;)
might consider rewriting it with fullpage..

**Comment by alvarotrigo** [comment]
Looking good! ;)
